presto-verifier/src/main/java/com/facebook/presto/verifier/checksum/ArrayColumnValidator.java

```diff
@@ -16,56 +16,86 @@
 import com.facebook.presto.common.type.ArrayType;
 import com.facebook.presto.common.type.RowType;
 import com.facebook.presto.common.type.Type;
-import com.facebook.presto.sql.tree.CoalesceExpression;
-import com.facebook.presto.sql.tree.Expression;
-import com.facebook.presto.sql.tree.FunctionCall;
-import com.facebook.presto.sql.tree.LongLiteral;
-import com.facebook.presto.sql.tree.QualifiedName;
-import com.facebook.presto.sql.tree.SingleColumn;
-import com.facebook.presto.sql.tree.TryExpression;
+import com.facebook.presto.sql.tree.*;
```

Comment:
don't use * imports

Reply:
These are added automatically by the IJ, I think, will remove, but got to keep an eye on it.